_tutorials/2016-05-10-iosxr-ansible.md

```diff
@@ -2,35 +2,41 @@
 permalink: "/tutorials/IOSXR-Ansible"
 author: Mike Korshunov
 excerpt: Getting started with IOSXR and Ansible playbooks
-published: true
+published: false
 title: "IOS-XR Ansible"
 date: "2016-05-16 01:18 +0530"
 ---
+
+{% include toc icon="table" title="IOS-XR Ansible" %}
+
+
+
 # Introduction
-In this tutorial we are going to cover XRv4 configuration with Ansible. We will setup and try simplest configuration with Unix machine connected to XRv64.
+In this tutorial we are going to cover IOS-XRv configuration with Ansible.We will set up and try a very simple configuration with a linux machine connected to IOS-XRv.
+
 
 ## Prerequisites
 - Computer with with 4-5GB free RAM;
 - Vagrant;
 - Ansible;
-- [XRv64 image](http://engci-maven-master.cisco.com/artifactory/appdevci-snapshot/);
+- [IOS-XRv image](http://engci-maven-master.cisco.com/artifactory/appdevci-snapshot/);
+- [Vagrantfile and scripts for provisioning](https://cisco.box.com/s/wykoquj1z76gbedq2mj0pgt0sfhwjvhv)
 
 ### Vagrant pre-setup
 
-To add XR box, image should be downloaded, after this issue the command:
-	
-    $ vagrant box add xrv64 iosxrv-fullk9-x64.box_2016-05-07-19-04-50
+Setup was tested on Windows, but it's the same for other OS. To add XR box, image should be downloaded, after this issue the command:
```

Comment:
Not particularly problematic, but call it a box instead of an image to avoid confusion with ISOs that Cisco customers were used to in the past.